### PR TITLE
s3: Add Liara LOS to provider list

### PR DESCRIPTION
**Pull request by vashian**
#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-new-feature-or-bug-fix).
- [x] I have added tests for all changes in this PR if appropriate.
- [x] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

**Comment by vashian** [comment]
@ncw Liara Object Storage (**LOS**) is a cloud object storage service and is  compatible with S3.

**Review by 7qqr — APPROVED** [review]

File: docs/content/_index.md [hunk 1]
```diff
@@ -135,6 +135,7 @@ WebDAV or S3, that work out of the box.)
 {{< provider name="IDrive e2" home="https://www.idrive.com/e2/" config="/s3/#idrive-e2" >}}
 {{< provider name="IONOS Cloud" home="https://cloud.ionos.com/storage/object-storage" config="/s3/#ionos" >}}
 {{< provider name="Koofr" home="https://koofr.eu/" config="/koofr/" >}}
+{{< provider name="Liara Object Storage (LOS)" home="https://liara.ir/landing/%D9%87%D8%A7%D8%B3%D8%AA-%D8%AF%D8%A7%D9%86%D9%84%D9%88%D8%AF-download-host" config="/s3/#liara-object-storage-los" >}}
```
Comment (albertony) [inline docs/content/_index.md:138]: The home url here should be a regular human-readable web page, and I think the current is an API endpoint?

File: docs/content/s3.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ The S3 backend can be used with a number of different providers:
 {{< provider name="IBM COS S3" home="http://www.ibm.com/cloud/object-storage" config="/s3/#ibm-cos-s3" >}}
 {{< provider name="IDrive e2" home="https://www.idrive.com/e2/" config="/s3/#idrive-e2" >}}
 {{< provider name="IONOS Cloud" home="https://cloud.ionos.com/storage/object-storage" config="/s3/#ionos" >}}
+{{< provider name="Liara Object Storage (LOS)" home="https://liara.ir/landing/%D9%87%D8%A7%D8%B3%D8%AA-%D8%AF%D8%A7%D9%86%D9%84%D9%88%D8%AF-download-host" config="/s3/#liara-object-storage-los" >}}
```
Comment (albertony) [inline docs/content/s3.md:23]: The home url here should be a regular human-readable web page, and I think the current is an API endpoint?

File: docs/content/s3.md [hunk 3]
```diff
@@ -641,7 +642,7 @@ A simple solution is to set the `--s3-upload-cutoff 0` and force all the files t
 {{< rem autogenerated options start" - DO NOT EDIT - instead edit fs.RegInfo in backend/s3/s3.go then run make backenddocs" >}}
```
Comment (albertony) [inline docs/content/s3.md:642]: Did you notice this? Its easy to miss, so I just wanted to make sure you did not manually edit text below here - which are not also reflected in the s3.go source and would therefore be lost.

**Review by albertony — CHANGES_REQUESTED** [review]

File: docs/content/_index.md (hunk 1 above)
Comment (ncw) [reply docs/content/_index.md:138]: Note that this bit `%D9%87%D8%A7%D8%B3%D8%AA-%D8%AF%D8%A7%D9%86%D9%84%D9%88%D8%AF` is UTF-encoded text which displays as `هاست-دانلود`.

"download host" seems to be the name of the service in English and with google translate the page does talk about s3 interfaces, its just quite a long way down the page.

However this page looks like it is the target for a Advertising campaign and not the main info page so I'm not sure this is the right page to link to?

I found this page which is the docs page https://docs.liara.ir/buckets/about but we usually link to the providers main page about the service.

Comment (mhemrg) [reply docs/content/_index.md:138]: This is the landing page of the product written in Farsi.
Liara is a cloud provider and Object Storage is one of their services.

**Review by ncw — COMMENTED** [review]
Please can you drop the changes to the autogenerated part of s3.md

I will merge after that - thank you :-)

**Comment by vashian** [comment]
Using the new URL `liara.ir/landing/object-storage`
Now it's human-readable without any encoded character.

Also, drop the changes from **autogenerated options** part.

**Comment by vashian** [comment]
@ncw Unfortunately I haven't heard from you in a while. Could you please tell me what went wrong?

**Comment by ncw** [comment]
@vashian sorry, I'm waiting for an update from you to remove the auto generated content from s3.md - otherwise your PR looks fine.  So remove all the changes in s3.md between

```
{{< rem autogenerated options start" - DO NOT EDIT - instead edit fs.RegInfo in backend/s3/s3.go then run make backenddocs" >}}
```

and

```
{{< rem autogenerated options stop >}}
```

These will be put in as part of the release process - I prefer not to add them at this point.

Thank you

**Comment by vashian** [comment]
> @vashian sorry, I'm waiting for an update from you to remove the auto generated content from s3.md - otherwise your PR looks fine. So remove all the changes in s3.md between
> 
> ```
> {{< rem autogenerated options start" - DO NOT EDIT - instead edit fs.RegInfo in backend/s3/s3.go then run make backenddocs" >}}
> ```
> 
> and
> 
> ```
> {{< rem autogenerated options stop >}}
> ```
> 
> These will be put in as part of the release process - I prefer not to add them at this point.
> 
> Thank you

@ncw I appreciate it. Thank you for your time.

**Review by albertony — APPROVED** [review]

**Review by ncw — APPROVED** [review]
That looks great - I'll merge now - thank you :-)

**Pull request merged**